api/test/round-tracker.test.js

```diff
@@ -623,7 +405,6 @@ describe('Round Tracker', () => {
           {
             cid: 'bafkqaaa',
             miner_id: 'f0010',
-            // Important: clients are deduplicated
```

Comment:
Please preserve this comment.

Reply:
Done https://github.com/CheckerNetwork/spark-api/pull/649/commits/e5cff37e4b87c5eac8652d9495827021104dff77